MekHQ/src/mekhq/gui/dialog/RetirementDefectionDialog.java

```diff
@@ -590,7 +590,7 @@ public boolean include(Entry<? extends RetirementTableModel, ? extends Integer>
                         && !rdTracker.getRetirees(contract).contains(person.getId())) {
                     return false;
                 } else {
-                    return nGroup.getFilteredInformation(person);
+                    return nGroup.getFilteredInformation(person,hqView.getCampaign().getLocalDate());
```

Comment:
Style change request: spaces between parameters in parameter lists for readability.